pkg/ibmvpc/routing_tables_analysis.go

```diff
@@ -385,52 +439,98 @@ func (rt *routingTable) computeDisjointRouting() error {
 	})
 
 	destIPBlocks := []*ipblock.IPBlock{}
-	for _, route := range rt.routesList {
+	for _, route := range routesList {
 		destIPBlocks = append(destIPBlocks, route.destIPBlock)
 	}
 	disjointDestinations := ipblock.DisjointIPBlocks(destIPBlocks, destIPBlocks)
 	for _, disjointDest := range disjointDestinations {
 		logging.Debugf("disjoint dest: %s\n", disjointDest.ToIPRanges())
-		if err := rt.computeRoutingForDisjointDest(disjointDest); err != nil {
-			return err
+		if err := res.computeRoutingForDisjointDest(routesList, disjointDest); err != nil {
+			return nil, err
 		}
 	}
-	return nil
+	return res, nil
 }
 
-func (rt *routingTable) getEgressPath(src vpcmodel.Node, dest *ipblock.IPBlock) (vpcmodel.Path, error) {
-	path, shouldDelegate := rt.getPath(dest)
+// If subnets are attached to the route's routing table, egress traffic from those
+// subnets in this zone will be subject to this route
```

Comment:
I don't understand this comment. Subnets are attached to a routing table, not to a specific route

Reply:
updated the comment